spanner/opentelemetry/src/main/java/com/example/spanner/OpenTelemetryUsage.java

```diff
@@ -0,0 +1,140 @@
+
+    // Create a new meter provider
+    SdkMeterProvider sdkMeterProvider = SdkMeterProvider.builder()
+        // Use Otlp exporter or any other exporter of your choice.
+        .registerMetricReader(
+            PeriodicMetricReader.builder(OtlpGrpcMetricExporter.builder().build()).build())
+        .build();
+
+    // Create a new tracer provider
+    SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
+        // Use Otlp exporter or any other exporter of your choice.
+        .addSpanProcessor(SimpleSpanProcessor.builder(OtlpGrpcSpanExporter
+            .builder().build()).build())
+            .build();
+
+    // Configure OpenTelemetry object using Meter Provider and Tracer Provider
+    OpenTelemetry openTelemetry = OpenTelemetrySdk.builder()
+        .setMeterProvider(sdkMeterProvider)
+        .setTracerProvider(sdkTracerProvider)
+        .build();
+
+    // Inject OpenTelemetry object via Spanner options or register as GlobalOpenTelemetry.
+    SpannerOptions options = SpannerOptions.newBuilder()
+        .setOpenTelemetry(openTelemetry)
+        .build();
+    Spanner spanner = options.getService();
+
+    // [END spanner_opentelemetry_usage]
+    DatabaseClient dbClient = spanner
+        .getDatabaseClient(DatabaseId.of(projectId, instanceId, databaseId));
+
+    captureQueryStatsMetric(openTelemetry, dbClient);
+    captureGfeMetric(dbClient);
+    sdkMeterProvider.forceFlush();
+    sdkTracerProvider.forceFlush();
+  }
+
+
+  // [START spanner_opentelemetry_capture_query_stats_metric]
+  static void captureQueryStatsMetric(OpenTelemetry openTelemetry, DatabaseClient dbClient) {
+    // Register query stats metric.
+    Meter meter = openTelemetry.getMeter("cloud.google.com/java");
+    DoubleHistogram queryStatsMetricLatencies =
+        meter
+            .histogramBuilder("spanner/query_stats_elapsed")
+            .setDescription("The execution of the query")
+            .setUnit("ms")
+            .build();
+
+    // Capture query stats metric data.
+    try (ResultSet resultSet = dbClient.singleUse()
+        .analyzeQuery(Statement.of("SELECT SingerId, AlbumId, AlbumTitle FROM Albums"),
+            QueryAnalyzeMode.PROFILE)) {
+
+      while (resultSet.next()) {
+        System.out.printf(
+            "%d %d %s", resultSet.getLong(0), resultSet.getLong(1), resultSet.getString(2));
+      }
+
+      String value = resultSet.getStats().getQueryStats()
+          .getFieldsOrDefault("elapsed_time", Value.newBuilder().setStringValue("0 msecs").build())
+          .getStringValue();
+      double elapsedTime = value.contains("msecs")
+          ? Double.parseDouble(value.replaceAll(" msecs", ""))
+          : Double.parseDouble(value.replaceAll(" secs", "")) * 1000;
+      queryStatsMetricLatencies.record(elapsedTime);
+    }
+  }
+  // [END spanner_opentelemetry_capture_query_stats_metric]
+
+  // [START spanner_opentelemetry_gfe_metric]
+  static void captureGfeMetric(DatabaseClient dbClient) {
```

Comment:
For someone new it isn't immediately obvious where the GFE metric is being captured. May be add a comment about the GFE metric name and that it is captured automatically if OT metrics are enabled.